src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -270,296 +281,27 @@ internal static void AddFunctionError(Dictionary<string, Collection<string>> fun
         }
 
         /// <summary>
-        /// Performs all required initialization on the host. Must be called before the host is started.
+        /// Performs all required initialization on the host.
+        /// Must be called before the host is started.
         /// </summary>
         public void Initialize()
         {
-            FileUtility.EnsureDirectoryExists(ScriptConfig.RootScriptPath);
-            string hostLogPath = Path.Combine(ScriptConfig.RootLogPath, "Host");
-            FileUtility.EnsureDirectoryExists(hostLogPath);
-            string debugSentinelFileName = Path.Combine(hostLogPath, ScriptConstants.DebugSentinelFileName);
-            this.LastDebugNotify = File.GetLastWriteTime(debugSentinelFileName);
-
-            FunctionTraceWriterFactory = new FunctionTraceWriterFactory(ScriptConfig);
-
-            IMetricsLogger metricsLogger = CreateMetricsLogger();
-
-            using (metricsLogger.LatencyEvent(MetricEventNames.HostStartupLatency))
+            _stopwatch.Start();
```

Comment:
I was using this stopwatch (and the logs it emits - see later below) during some local performance testing. Was going to remove, but now I wonder if we shouldn't just leave this in here. The logs are useful.